src/flynt/cli.py

```diff
@@ -117,7 +126,9 @@ def run_flynt_cli():
         parser.print_usage()
         return 1
 
-    print(f"Running flynt v.{__version__}")
+    if not args.string:
```

Comment:
Should this check for `not args.quiet` as well?

Reply:
probably it should.